src/test/resources/pages/InstructorHomeHTML.html

```diff
@@ -1,241 +1,318 @@
 <div id="frameBodyWrapper" class="container theme-showcase">
```

Comment:
There should be no changes to html pages. If html pages have to change, we need to do that change first as a separate issue before converting to jstl.